pandas/tests/arrays/categorical/test_operators.py

```diff
@@ -395,3 +395,20 @@ def test_numeric_like_ops(self):
         msg = "Object with dtype category cannot perform the numpy op log"
         with pytest.raises(TypeError, match=msg):
             np.log(s)
+
+    def test_contains(self, ordered):
+        # GH-37867
+        cat = Categorical(["a", "b"], ordered=ordered)
+        assert "a" in cat
+        assert "x" not in cat
+        assert pd.NA not in cat
+
+        cat = Categorical([np.nan, "a"], ordered=ordered)
+        assert "a" in cat
+        assert "x" not in cat
+        assert pd.NA in cat
```

Comment:
can you tests `np.nan` here as well (True), while `pd.NaT` will be False

Reply:
Shouldn't `pd.NA in cat` not give False? Since it is np.nan that Categorical is using as missing value indicator, and not pd.NA

Reply:
In the current implementation, they all give True, so `pd.NaT in cat` gives True also, This is because `cat` is object-dtype, so can contains a `pd.NaT`.